domain/schema/model/sql/0020-unit.sql

```diff
@@ -52,6 +52,17 @@ ON unit (application_uuid);
 CREATE INDEX idx_unit_net_node
 ON unit (net_node_uuid);
 
+CREATE TABLE unit_channel (
+    unit_uuid TEXT NOT NULL,
+    track TEXT,
+    risk TEXT NOT NULL,
+    branch TEXT,
+    CONSTRAINT fk_unit_origin_unit
```

Comment:
```suggestion
    CONSTRAINT fk_unit_channel_unit
```

Reply:
Changed the table so no longer relevant.